netengine.py

```diff
@@ -21,11 +64,21 @@ def get_current_machine_config():
     current_machine = config.machines[CURRENT_IP]
     return current_machine
 
+def validate_language(lang):
```

Comment:
can we change this to `is_supported_language`? I like question like naming for boolean return methods.